blueprints/app/files/tests/helpers/module-for-acceptance.js

```diff
@@ -11,12 +11,12 @@ export default function(name, options = {}) {
       this.application = startApp();
 
       if (options.beforeEach) {
-        return options.beforeEach.apply(this, arguments);
+        return options.beforeEach.apply(...arguments);
```

Comment:
I feel like this conversation has happened before... but you're obviously right!